docs/en/supported_models/supported_models.md

```diff
@@ -2,32 +2,34 @@
-|       QWen-VL       |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
-|     DeepSeek-VL     |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
-|      Baichuan       |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
-|      Baichuan2      |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
-|     Code Llama      |   7B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
-|         YI          |   6B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
-|   LLaVA(1.5,1.6)    |   7B - 34B   |    Yes    |   Yes   |   Yes   |  Yes  |
-|    InternVL-Chat    |  v1.1- v1.5  |    Yes    |   Yes   |   Yes   |  Yes  |
-|       MiniCPM       | Llama3-V-2_5 |    Yes    |   Yes   |   Yes   |  Yes  |
-|   MiniGeminiLlama   |      7B      |    Yes    |   No    |   No    |  Yes  |
-|        GLM4         |      9B      |    Yes    |   Yes   |   Yes   |  No   |
-|      CodeGeeX4      |      9B      |    Yes    |   Yes   |   Yes   |  No   |
+|         Model         |     Size     | FP16/BF16 | KV INT8 | KV INT4 | W4A16 |
+| :-------------------: | :----------: | :-------: | :-----: | :-----: | :---: |
+|         Llama         |   7B - 65B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|        Llama2         |   7B - 70B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|        Llama3         |   8B, 70B    |    Yes    |   Yes   |   Yes   |  Yes  |
+|       InternLM        |   7B - 20B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|       InternLM2       |   7B - 20B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|      InternLM2.5      |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|  InternLM-XComposer   |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|  InternLM-XComposer2  | 7B, 4khd-7B  |    Yes    |   Yes   |   Yes   |  Yes  |
+| InternLM-XComposer2.5 |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|         QWen          |  1.8B - 72B  |    Yes    |   Yes   |   Yes   |  Yes  |
+|        QWen1.5        | 1.8B - 110B  |    Yes    |   Yes   |   Yes   |  Yes  |
+|         QWen2         |  1.5B - 72B  |    Yes    |   Yes   |   Yes   |  Yes  |
+|        Mistral        |      7B      |    Yes    |   Yes   |   Yes   |  No   |
+|        QWen-VL        |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|      DeepSeek-VL      |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|       Baichuan        |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|       Baichuan2       |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|      Code Llama       |   7B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
+|          YI           |   6B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
+|    LLaVA(1.5,1.6)     |   7B - 34B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|     InternVL-Chat     |  v1.1- v1.5  |    Yes    |   Yes   |   Yes   |  Yes  |
+|       InternVL2       |    1B-40B    |    Yes    |   Yes   |   Yes   |  Yes  |
```

Comment:
InternVL2 1B is not compatible with TurboMind but is supported by the PyTorch Engine. Perhaps we may clarify this.

Reply:
It has two tables.